### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by darkledarkle**
help

### CODE REVIEW

**Review by nytrinh — CHANGES_REQUESTED** [review]
its 2022 u silly

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+2018 robot is borked so we need to update everything
```
Comment (darkledarkle) [inline README.md:1]: its wrong

**Review by darkledarkle — COMMENTED** [review]
COOL!

**Review by darkledarkle — COMMENTED** [review]
its not fixed....

File: README.md (hunk 1 above)
Comment (eggliot) [reply README.md:1]: fix it

**Review by eggliot — CHANGES_REQUESTED** [review]

**Pull request closed without merging**